lib/topological_inventory/scheduler/database_tools.rb

```diff
@@ -0,0 +1,28 @@
+require 'topological_inventory/scheduler/logging'
+require 'active_support/core_ext/integer/time'
+
+module TopologicalInventory
+  module Scheduler
+    class DatabaseTools
+      include Logging
+
+      REFRESH_STATES_TIME_THRESHOLD = 1.week.freeze
+
+      def self.remove_old_records
+        threshold = DateTime.now - REFRESH_STATES_TIME_THRESHOLD  
+
+        TopologicalInventory::Scheduler::logger.info(
```

Comment:
you can use just `logger.info()`

Reply:
then you have to use `extend Logging`, or (I prefer to be consistent with other services) change this method to object method